index.html

```diff
@@ -0,0 +1,328 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- CSS only -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css"
```

Comment:
краще тягнути тільки **grid**, бо решту ми не використовуємо
```suggestion
      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap-grid.min.css"
```